dockstore-client/src/main/java/io/dockstore/client/cli/nested/WesCommandParser.java

```diff
@@ -105,4 +108,22 @@ public boolean isVerbose() {
     public static class CommandServiceInfo extends WesMain {
     }
 
+    @Parameters(separators = "=", commandDescription = "Retrieve a list of runs")
```

Comment:
Looked into the jCommander docs, and it looks like the `separators` parameter is supposed to override the default space separator. Interestingly, both `--count 3` and `--count=3` worked as expected. To avoid any confusion, I removed `=` from the separator list: https://github.com/dockstore/dockstore-cli/pull/112/commits/604833e31d6f0809c7d5353fa74a58dfaf4008dd